templates/repo/issue/view_content/pull.tmpl

```diff
@@ -164,7 +164,7 @@
 				{{$notAllOverridableChecksOk := or .IsBlockedByApprovals .IsBlockedByRejection .IsBlockedByOfficialReviewRequests .IsBlockedByOutdatedBranch .IsBlockedByChangedProtectedFiles (and .EnableStatusCheck (not .RequiredStatusCheckState.IsSuccess))}}
 
 				{{/* admin can merge without checks, writer can merge when checks succeed */}}
```

Comment:
```suggestion
				{{/* admin can merge without checks, writer can merge when checks succeed, BlockAdminMergeOverride only hides the button on web UI but doesn't stop the admin use API or web request to force merging */}}
```

I think this change means "UI-only hidden", but not completely prevent admin users from merging.

Reply:
Thanks for the pointer. I'll look into changing the API endpoints for merging pull requests.